enclone_print/src/print_clonotypes.rs

```diff
@@ -359,7 +345,7 @@ pub fn print_clonotypes(
             // Generate Loupe data.
 
             if (!ctl.gen_opt.binary.is_empty() || !ctl.gen_opt.proto.is_empty()) && pass == 2 {
-                loupe_clonotypes.push(make_loupe_clonotype(
```

Comment:
Ah, this was always a vector with 0 or 1 elements, so we changed it to an option. Makes sense

Reply:
Yeah, there are a lot of places in this codebase where a vector is used as a proxy for optionality.  It is confusing at first but ends up making way more sense once refactored to an option, especially in cases where several vectors represent several items that are all either present or absent together.  I have more clean-up along these lines in subsequent patches.

Reply:
This section of code is kind of confusing because it has this two-pass loop with a lot of pass-wise conditionals.  I'm unrolling this in follow-up code to make it easier to understand what's happening.